umh-core/cmd/main.go

```diff
@@ -53,13 +54,61 @@ func main() {
 
 	// Load the config
 	configManager := config.NewFileConfigManager()
-	config, err := configManager.GetConfig(ctx, 0)
+	configData, err := configManager.GetConfig(ctx, 0)
 	if err != nil {
 		sentry.ReportIssuef(sentry.IssueTypeFatal, log, "Failed to load config: %s", err)
 	}
 
+	// Extract environment variables and apply them to the config
```

Comment:
maybe some words here how this works. that this is used for initial filling of the config, or to overwrite certain parameters. 

it makes sense to also explain all parameters and env variables here what they are doing. so we can alter generate a good README for that. "So AUTH_TOKEN will set set/overwrite the auth-token in the config file. Useful for initializing the config file"